index.html

```diff
@@ -51,5 +51,201 @@ <h3 class="connect">LET'S CONNECT</h3>
+          <li class="canopy">CANOPY</li>
+          <li><img src="images/Counter.png" alt="counter" /></li>
+          <li class="light">Back End Dev</li>
+          <li><img src="images/Counter.png" alt="counter" /></li>
+          <li class="light">2015</li>
+        </ul>
+        <p class="description-2 flex0">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required.
+        </p>
+        <ul class="ul-box2 flex0">
+          <li class="highlight font-1">html</li>
+          <li class="highlight font-2">css</li>
+          <li class="highlight font-3">javaScript</li>
+        </ul>
+        <a href="#" class="highlight-2 font-4 flex0"> See Project </a>
+      </div>
+      <div class="card card-4">
+        <a href="#"
+          ><img
+            src="images/Snapshoot-Portfolio3.png"
+            alt="Snapshoot-Portfolio"
+            class="snapshot"
+        /></a>
+        <p class="tonic flex0">Multi-Post Stories</p>
+        <ul class="ul-box1 flex0">
+          <li class="canopy">CANOPY</li>
+          <li><img src="images/Counter.png" alt="counter" /></li>
+          <li class="light">Back End Dev</li>
+          <li><img src="images/Counter.png" alt="counter" /></li>
+          <li class="light">2015</li>
+        </ul>
+        <p class="description-2 flex0">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required.
+        </p>
+        <ul class="ul-box2 flex0">
+          <li class="highlight font-1">html</li>
+          <li class="highlight font-2">css</li>
+          <li class="highlight font-3">javaScript</li>
+        </ul>
+        <a href="#" class="highlight-2 font-4 flex0"> See Project </a>
+      </div>
+    </section>
+    <section class="about-myself">
+      <div class="container-3">
+        <div class="top">
+          <h1 class="first-title-2">About Myself</h1>
+          <p class="description-3">
+            Hello I’m a software developer! I can help you build a product ,
+            feature or website Look through some of my work and experience! If
+            you like what you see and have a project you need coded, don’t
+            hestiate to contact me.
+          </p>
+          <h3 class="connect-2">LET'S CONNECT</h3>
+          <ul class="container-2">
+            <li>
+              <a href="#"> <img src="images/twiter.svg" alt="Twitter" /></a>
+            </li>
+            <li>
+              <a href="#"> <img src="images/Linkedin.svg" alt="Linkedin" /></a>
+            </li>
+            <li>
+              <a href="#"> <img src="images/m.svg" alt="m" /></a>
+            </li>
+            <li>
+              <a href="#"> <img src="images/github.svg" alt="github" /></a>
+            </li>
+            <li>
+              <a href="#"> <img src="images/manito.svg" alt="manito" /></a>
+            </li>
+          </ul>
+          <a href="#" class="highlight-3 font-5"> Get my resume </a>
```

Comment:
- [ ] Please let's follow the Figma template as much as possible this includes the buttons interactions.

## Figma
![image](https://user-images.githubusercontent.com/28109626/148601480-fdfa2ff6-72b6-4a65-a274-6a13f3056225.png)

## Your Project 
![image](https://user-images.githubusercontent.com/28109626/148601603-5dc6cf09-3566-4e0a-9da2-521607d68bca.png)